src/components/MoviesList/ResultListItem.tsx

```diff
@@ -0,0 +1,90 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+import { MoviesSearchResult } from "../../app/types";
+import { Image } from "../Image/Image";
+import { Ratings } from "../Ratings/Ratings";
+
+const Item = styled.li`
+  .highlighted {
+    color: ${props => props.theme.highlighted};
+  }
+  border-bottom: 1px solid ${props => props.theme.textColorSecondary};
+  width: 90%;
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+  padding: 15px 10px;
+  height: 100px;
+`;
+
+const Row = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+`;
+const RatingsStyled = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const MovieInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  a {
+    text-decoration: none;
+  }
+`;
+
+const MovieName = styled.h3`
+  margin: 0;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: ${props => props.theme.textColorPrimary};
+  margin-bottom: 5px;
+`;
+
+const Release = styled.span`
+  margin: 0;
+  font-weight: 400;
+  font-size: 14px;
+  color: ${props => props.theme.textColorSecondary};
+`;
+
+export function ResultListItem({ movie }: { movie: MoviesSearchResult }) {
```

Comment:
What do you think about using arrow functions for components? 🙂 
```suggestion
export const ResultListItem = ({ movie }: Props) => {
```

Reply:
agree, changed to arrow functions. :)